script.js

```diff
@@ -161,7 +161,11 @@ function handleData([sheetsText, statesGeoJson]) {
     }, new Map());
 
   const localitiesData = rows.filter(
-    row => row.admin_scale !== "State" && row.lat !== null && row.lon !== null
+    row => row.admin_scale !== "State" && row.lat !== null && row.lon !== null && row.admin_scale !=="County"
+  );
+  console.log(localitiesData);
+  const countiesData = rows.filter(
```

Comment:
@goldfarb same for here, just filter on `row.admin_scale === "County"` and drop the two `!==` for "City" and "State".